packages/jdk15.rb

```diff
@@ -0,0 +1,58 @@
+require 'package'
+
+class Jdk15 < Package
+  description 'The JDK is a development environment for building applications, applets, and components using the Java programming language.'
+  homepage 'https://www.oracle.com/java/technologies/javase-jdk15-downloads.html'
+  version '15.0.1'
+  compatibility 'x86_64'
+  case ARCH
+  when 'x86_64'
+    source_url 'file:///dev/null'
+    source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  end
+
+  binary_url ({
+    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/jdk15-15.0.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    x86_64: '456f4cca19b0e5bbc89cba06b80527861647fd7f6e740b476cd2cd86cefaacdd',
+  })
+
+  case ARGV[0]
+  when 'install', 'reinstall'
+    abort "JDK8 installed.".lightgreen if Dir.exists? "#{CREW_PREFIX}/share/jdk8"
```

Comment:
Can we use `exit(0)` to replace `abort`? `abort` will make the exit code non-zero and immediately stop the installation when installing this package as a dependency.

Reply:
It's fine to have an exit code of non-zero.  The idea here is to "abort" if any other version is installed.  So, in essence, this really means only one version of the jdk can be installed at a time.  In theory, other packages that depend on jdk8 could still work since the executables are the same.

Reply:
I mean the `abort` will exit the `package.rb` (not only the `jdk15.rb`), this will also cause packages (which depends on `jdk15`) installation stop.
e.g
```ruby
require 'jdk15'
# The installation will exit here due to the package.rb stopped
require 'vim'

def self.install
   ...
```

Reply:
That is fine.  The way to fix that would be to remove jdk8 or jdk11 and then try again.